### ISSUE TITLE
Since 3.5 update

### ISSUE THREAD

**Issue by veiszgabor79** [mattzzw/midi_muppet_hx_2btn#2]
Maybe the issue appeared earlier than the 3.5 firmware update on HX Stomp, but now it's present for sure. Some functions disappeared, like in the footswitch mode, the long press snapshot mode on the right pedal is not working - left pedal long press tuner is fine. In the scroll mode if you are in a patch, where you have assigned functions in the command center for the 3 pedals, scrolling up/down won't work. If you have set in the command center functions for only two pedals, left pedal won't work, right pedal will preset up.

**Comment by mattzzw** [comment]
Hm, that is strrange. Therw was no major change regarding CC71 which is used to switch from stomp mode to snapshot mode. The only noteworthy change regarding midi in the 3.5 update is this "known issue":

```In some cases, sending MIDI CC49-59 (footswitch emulation control) to engage stomp switches assigned to snapshot commands can result in inconsistent behavior. Instead, send Helix CC69 messages (values 0-7) to recall snapshots```

Midi muppet is using CC69 so that shouldn't be an issue.

Depending on which version of the code you are using, combining patch up/down with command center assignments could potentially cause issues as I used a kind of hack by switching into scroll mode and using pedal press emulation. Since firmware 3.1 I switched to using CC72 to step up/down patches. Check out lines 484 in the code. This should not interfere with pedal assignments for control center.

Right now I don't have a midi muppet any more so I am not able to reproduce or confirm this bug. But maybe I could give you some ideas where to look.

**Comment by veiszgabor79** [comment]
Dear Mattzzw,
thanks for the very fast answer. 
I tried to look for the latest code, and upload, but anyway I have problems with code uploading. Previously I tried to upload the code again, but there is an error message, that the board doesn't respond, so I had a spare Arduino board, which worked, I replaced the old one with that. But this new one doesn't want to work this time again, as the old one did. Maybe it's a really silly user error, I'm very unexperienced in this whole programming stuff. Anyway, it works in a way I still can use it. :)
